packages/turf-clean-coords/index.js

```diff
@@ -0,0 +1,109 @@
+var clone = require('@turf/clone');
+var invariant = require('@turf/invariant');
+var getCoords = invariant.getCoords;
+var getGeomType = invariant.getGeomType;
+
+/**
+ * Removes redundant coordinates from a (Multi)LineString or (Multi)Polygon; ignores (Multi)Point.
+ *
+ * @name cleanCoords
+ * @param {Geometry|Feature<any>} geojson Feature or Geometry
```

Comment:
I've been dropping the `<any>` in the recent JSDocs.
`{Geometry|Feature<any>}` => `{Geometry|Feature}`